docs/iot/iot-partners/anchor/anchor.md

```diff
@@ -0,0 +1,73 @@
+# Tizen IoT with Anchor SoM Module
+
+> The Fastest Way to design your IoT Applications
```

Comment:
How about using simple sentence?

Just "The Fastest Way to design your IoT Applications" with \>.